apps/api/v2/src/modules/organizations/services/organizations.service.ts

```diff
@@ -1,12 +1,24 @@
 import { OrganizationsRepository } from "@/modules/organizations/organizations.repository";
+import { OrganizationUsersRepository } from "@/modules/organizations/repositories/organization-users.repository";
 import { Injectable } from "@nestjs/common";
 
 @Injectable()
 export class OrganizationsService {
-  constructor(private readonly organizationsRepository: OrganizationsRepository) {}
+  constructor(
+    private readonly organizationsRepository: OrganizationsRepository,
+    private readonly organizationUsersRepository: OrganizationUsersRepository
+  ) {}
 
   async isPlatform(organizationId: number) {
     const organization = await this.organizationsRepository.findById(organizationId);
     return organization?.isPlatform;
   }
+
+  async getOrganizationUsers(organizationId: number, emailInput?: string | string[]) {
```

Comment:
I think we should enforce emailInput to be an array in this function, never just a string, validating and transforming the data to be useable by the rest of the api should be done inside of classValidators, see @Transform for an example within create-booking.input.ts